### PR TITLE
Add scipy subdirectory to dylib deletion

### PR DESCRIPTION
**Pull request by rosteen**
Fingers crossed.

### CODE REVIEW

**Comment by rosteen** [comment]
@pllim I don't recall, was there a reason for not making the `Build standalone` label run the MacOS build or was that an oversight when the jobs were split?

**Comment by pllim** [comment]
I answered in tag-up. It was to do with some secret not accessible from fork, so it is intentional.

File: .github/workflows/standalone.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ jobs:
   # Do not want to deal with OSX certs in pull request builds.
   build_binary_osx:
     runs-on: ${{ matrix.os }}-latest
-    if: (github.repository == 'spacetelescope/jdaviz' && (github.event_name == 'push' || github.event_name == 'workflow_dispatch'))
+    if: (github.repository == 'spacetelescope/jdaviz' && (github.event_name == 'push' || github.event_name == 'workflow_dispatch' || || contains(github.event.pull_request.labels.*.name, 'Build standalone')))
```
Comment (pllim) [inline .github/workflows/standalone.yml:80]: This needs to be reverted.

Comment (rosteen) [reply .github/workflows/standalone.yml]: Done.

**Review by pllim — APPROVED** [review]
Thanks!

**Comment by pllim** [comment]
@rosteen , please review https://github.com/spacetelescope/jdaviz/pull/2928 . Thanks!

**Pull request merged**